asm/opcode.go

```diff
@@ -18,22 +18,22 @@ type Class uint8
 const classMask OpCode = 0x07
 
 const (
-	// LdClass load memory
```

Comment:
Maybe we just copy what https://www.kernel.org/doc/html/latest/bpf/instruction-set.html#instruction-classes has?

Reply:
For LD, it states 'non-standard load operations', which is as vague as can be. Then IMO what's in this patch is clearer. :shrug:

Reply:
But the current description doesn't capture the quirkiness of LoadIndOp which loads from an sk_buff for cBPF compat.

Reply:
Done.